src/autodiscovery.js

```diff
@@ -308,11 +311,20 @@ export class AutoDiscovery {
                 `${isUrl}/_matrix/identity/api/v1`,
             );
             if (!isResponse || !isResponse.raw || isResponse.action !== "SUCCESS") {
+                logger.log("[Debug] Second error: response from IS failed");
```

Comment:
Also possible unnecessary.

Reply:
this one is, yes. I plan on just reverting this whole commit once I don't need it anymore, and am feeling a bit lazy to fix this